[core]/esx_multicharacter/server/main.lua

```diff
@@ -159,11 +159,12 @@ elseif ESX.GetConfig().Multichar then
 			local count = 0
 
 			for i = 1, #DB_COLUMNS do
-				local v = DB_COLUMNS[i]
+				local column = DB_COLUMNS[i]
+				DB_TABLES[column.TABLE_NAME] = column.COLUMN_NAME
```

Comment:
DB_TABLES where use? I don't know what is this.

Reply:
> DB_TABLES where use? I don't know what is this.

its used for deleting characters. it loops through, the tables and deletes all the user data from the database. you probably dont recognise it because its been broken like this since the commit where it got added into the main repo, during that transition there was an update that accidently broke it. ive known about it for ages, its just been too simple of a fix for me to put the effort into PRing :)

Reply:
I understand there is no better way to do this? If everything is already linked to everything and we know what table the player data can be in, then delete it with a query?

Reply:
This is the best way to do this, yes. Any other way would require all scripts to have code in for deletion and would end up with next to no adoption from the community, which would make deleting a character useless, since all the character data would still be in the database 🥲